src/main/resources/org/springframework/integration/config/xml/spring-integration-kafka-1.2.xsd

```diff
@@ -117,6 +141,18 @@
 											</xsd:appinfo>
 										</xsd:annotation>
 									</xsd:attribute>
+									<xsd:attribute name="value-serializer" use="optional" type="xsd:string">
+										<xsd:annotation>
+											<xsd:appinfo>
+												<xsd:documentation>
+													Custom implementation of a Kafka Encoder for encoding message values.
```

Comment:
Wording - should refer to serializer, explain that options are mutually exclusive.